sample-apps/ruby-rails-sample-app/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,78 @@
+require 'net/http'
+
+def get_xray_trace_id(otel_trace_id_hex)
+    xray_trace_id = "1-#{otel_trace_id_hex[0..7]}-#{otel_trace_id_hex[8..otel_trace_id_hex.length]}"
+    { traceId: xray_trace_id }
+end
+
+class ApplicationController < ActionController::Base
```

Comment:
Consistency: in some files that were auto generated we are using 2 spaces for indentation. Here we are using 4. can you check what is the standard for ruby applications and follow in all files?

Reply:
2 space indentation is standard. I updated it